src/server.c

```diff
@@ -1372,6 +1372,10 @@ qd_server_t *qd_server(qd_dispatch_t *qd, int thread_count, const char *containe
 void qd_server_free(qd_server_t *qd_server)
 {
     if (!qd_server) return;
+
+    qd_http_server_stop(qd_server->http); /* Stop HTTP threads immediately */
+    qd_http_server_free(qd_server->http);
```

Comment:
the function qd_http_server_free() in http-libwebsockets.c seems to be calling qd_http_server_stop(), so just calling qd_http_server_free() is enough ?

Reply:
It appears that calling only `qd_http_server_free` is indeed sufficient. I was thinking that maybe calling _stop before _free is some sort of dispatch convention... Do you want me to remove the call to _stop here?

Reply:
Yes, let's remove the call to qd_http_server_stop(). Otherwise this PR is good to go. Thanks.